test/const/test_consts.go

```diff
@@ -0,0 +1,130 @@
+    "aadClientSecret": "{{.AADClientSecret}}",
+    "resourceGroup": "{{.ResourceGroup}}",
+    "location": "{{.Location}}",
+    "vmType": "{{.VMType}}"
+}`
+	DefaultAzurePublicCloudLocation = "eastus2"
+	DefaultAzurePublicCloudVMType   = "vmss"
+
+	// Env vars
+	TenantIDEnvVar        = "AZURE_TENANT_ID"
+	SubscriptionIDEnvVar  = "AZURE_SUBSCRIPTION_ID"
+	AadClientIDEnvVar     = "AZURE_CLIENT_ID"
+	AadClientSecretEnvVar = "AZURE_CLIENT_SECRET"
+	ResourceGroupEnvVar   = "AZURE_RESOURCE_GROUP"
+	LocationEnvVar        = "AZURE_LOCATION"
+	VMTypeEnvVar          = "AZURE_VM_TYPE"
+
+	VolumeSnapshotClassKind = "VolumeSnapshotClass"
+	VolumeSnapshotKind      = "VolumeSnapshot"
+	VolumePVCKind           = "PersistentVolumeClaim"
+	APIVersionv1beta1       = "v1beta1"
+	SnapshotAPIGroup        = "snapshot.storage.k8s.io"
+	SnapshotAPIVersion      = SnapshotAPIGroup + "/" + APIVersionv1beta1
+
+	KubeconfigEnvVar = "KUBECONFIG"
+	ReportDirEnvVar  = "ARTIFACTS"
+	DefaultReportDir = "/workspace/_artifacts"
+
+	testMigrationEnvVar     = "TEST_MIGRATION"
+	testWindowsEnvVar       = "TEST_WINDOWS"
+	CloudNameEnvVar         = "AZURE_CLOUD_NAME"
+	inTreeStorageClass      = "kubernetes.io/azure-disk"
+	buildV2Driver           = "BUILD_V2"
+	useOnlyDefaultScheduler = "USE_ONLY_DEFAULT_SCHEDULER"
+
+	ExecTimeout = 10 * time.Second
+	// Some pods can take much longer to get ready due to volume attach/detach latency.
+	SlowPodStartTimeout = 10 * time.Minute
+	// Description that will printed during tests
+	FailedConditionDescription = "Error status code"
+
+	Poll            = 2 * time.Second
+	PollLongTimeout = 5 * time.Minute
+	PollTimeout     = 10 * time.Minute
+
+	testTolerationKey   = "test-toleration-key"
+	testTolerationValue = "test-toleration-value"
+	testLabelKey        = "test-label-key"
+	testLabelValue      = "test-label-value"
+
+	MasterNodeLabel = "node-role.kubernetes.io/master"
+)
+
+var (
+	AzurePublicCloudSupportedStorageAccountTypes = []string{"Standard_LRS", "Premium_LRS", "StandardSSD_LRS"}
+	AzureStackCloudSupportedStorageAccountTypes  = []string{"Standard_LRS", "Premium_LRS"}
+
+	TestTaint = v1.Taint{
+		Key:    testTolerationKey,
+		Value:  testTolerationValue,
+		Effect: v1.TaintEffectNoSchedule,
+	}
+
+	TestLabel = map[string]string{
+		testLabelKey: testLabelValue,
+	}
+
+	TestAffinity = v1.Affinity{
+		NodeAffinity: &v1.NodeAffinity{
+			RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+				NodeSelectorTerms: []v1.NodeSelectorTerm{
+					{
+						MatchExpressions: []v1.NodeSelectorRequirement{
+							{
+								Key:      testLabelKey,
+								Operator: v1.NodeSelectorOpExists,
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	IsUsingInTreeVolumePlugin   = os.Getenv(AzureDriverNameVar) == inTreeStorageClass
+	IsTestingMigration          = os.Getenv(testMigrationEnvVar) != ""
+	IsWindowsCluster            = os.Getenv(testWindowsEnvVar) != ""
+	IsUsingCSIDriverV2          = os.Getenv(buildV2Driver) != ""
```

Comment:
In order to support the MCR pipeline build, PR #296 changes usage of the `BUILD_V2` environment variable. This check will need to be changed to `strings.EqualFold(os.Getenv(buildV2Driver), "true")`.

Reply:
Fixed